src/integrationTest/java/uk/gov/hmcts/reform/notificationservice/data/NotificationRepositoryTest.java

```diff
@@ -155,7 +167,7 @@ void should_exclude_notifications_from_other_service() {
         // then
         assertThat(notifications)
             .isNotEmpty()
-            .hasSize(2)
+            .hasSize(3)
             .usingElementComparatorIgnoringFields("createdAt", "processedAt")
             .containsExactlyInAnyOrder(
```

Comment:
Should be `containsExactly`. And there are still several calls to `containsExactlyInAnyOrder` - they all should be replaced with `containsExactly` where ordered result is expected.

Reply:
Done.